### PR TITLE
feat(Task-1B): As a Project Manager, already done exercises uploaded to the git repository

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

File: Ejercicio2/index-START.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>JS + CSS Clock</title>
+</head>
+<body>
+
+
```
Comment (eloydotco) [inline Ejercicio2/index-START.html]: Please, remove this extra empty spacing

File: Ejercicio2/index-START.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      transition: all 0.5s;
+      transition-timing-function: cubic-bezier(0.1,1,0.58,1);
+    }
+
+  </style>
+
+  <script>
+    const secondHand=document.querySelector('.second-hand');
+    const minHand=document.querySelector('.min-hand');
+    const hourHand=document.querySelector('.hour-hand');
```
Comment (eloydotco) [inline Ejercicio2/index-START.html]: this case is very simple, but in the real world we should check all variable before using it, and/or provide a fallback value, for example:
```
const hoursHand = document.querySelector('.hour-hand') || 0;
```
This way hoursHand always will have a valid number value, if you don't know about `||` ,`&&` and falsy/truthy values, please investigate them beucase they are really useful

**Review by eloydotco — APPROVED** [review]

File: Ejercicio2/index-START.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      transition: all 0.5s;
+      transition-timing-function: cubic-bezier(0.1,1,0.58,1);
+    }
+
+  </style>
+
+  <script>
+    const secondHand=document.querySelector('.second-hand') || 0;
+    const minHand=document.querySelector('.min-hand') || 0;
+    const hourHand=document.querySelector('.hour-hand') || 0;
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio2/index-START.html]: As Eloy said, place whitespace before and after the '='

File: Ejercicio2/index-START.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+    function setDate(){
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio2/index-START.html]: whe usually place a whitespace between  '()' and '{'

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**